libs/sdk-core/src/persist/migrations.rs

```diff
@@ -270,6 +270,28 @@ pub(crate) fn current_migrations() -> Vec<&'static str> {
         FROM old_swaps;
 
        DROP TABLE old_swaps;            
+       ",
+
+        // reverse_swaps      holds the immutable data
+        // reverse_swaps_info holds the cached data, which can be reconstructed by any client
+       "
+       CREATE TABLE IF NOT EXISTS reverse_swaps (
```

Comment:
This should be created on the sync db

Reply:
Done 371a123f61403d0b86770ddd39a44210e1db9da5